processing/data_collection/gazette/spiders/mt_cuiaba.py

```diff
@@ -0,0 +1,54 @@
+import json
+import time
+from dateparser import parse
+from datetime import datetime
+from scrapy import Request
+from scrapy.downloadermiddlewares.retry import RetryMiddleware
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+# based on the website
+FIRST_YEAR = 1967
+BASE_URL = "https://diariooficial.cuiaba.mt.gov.br/api/api/editions"
+
+
+class MtCuiabaSpider(BaseGazetteSpider):
+    TERRITORY_ID = 5103403
+    name = "mt_cuiaba"
+    allowed_domains = ["diariooficial.cuiaba.mt.gov.br"]
+    start_urls = ["https://diariooficial.cuiaba.mt.gov.br/"]
+
+    custom_settings = {
+        "DOWNLOAD_DELAY": 1,
+    }
+
+    def start_requests(self):
+        today = datetime.today()
+        final_year = today.year
+        for year in range(FIRST_YEAR, final_year + 1):
+            for month in range(1, 13):
+                date_url = f"{BASE_URL}/published/{year}/{month}"
+                yield Request(
+                    url=date_url,
+                    headers={
+                        "referer": "https://diariooficial.cuiaba.mt.gov.br/edicoes",
+                    },
+                )
```

Comment:
To iterate over dates, I think `rrule` will be more elegant and readable. Take a look in the web and in the spiders:

```
processing/data_collection/gazette/spiders/pe_recife.py
processing/data_collection/gazette/spiders/rj_nova_iguacu.py
processing/data_collection/gazette/spiders/ro_porto_velho.py
processing/data_collection/gazette/spiders/sp_guarulhos.py
```

Let us know if you cannot use it.